equipment_handler/source/housekeeping_athena.c

```diff
@@ -41,17 +43,23 @@ int HAL_get_temp_all(long *temparray) {
  * @brief
  *      Getter to Athena uptime
  * @return
- * 		  integer OBC_uptime to store Athena uptime
+ * 		integer OBC_uptime to store Athena uptime
  *      Seconds = OBC_uptime*10
  *      Max = 655350 seconds (7.6 days)
  */
 uint16_t Athena_get_OBC_uptime() {
-    TickType_t OBC_ticks = xTaskGetTickCount(); // 1 tick/ms
-    uint32_t OBC_uptime_s = (OBC_ticks * portTICK_PERIOD_MS) / 1000;
 
-    // convert OBC_uptime from 32 bit to 16 bit. Max = 655350 seconds (7.6 days)
-    // Unit will be deca-seconds due to size restraint, seconds = OBC_uptime*10
-    return (uint16_t)OBC_uptime_s / 10;
+  TickType_t OBC_ticks = xTaskGetTickCount(); //1 tick/ms
+  uint32_t OBC_uptime_32 = OBC_ticks/1000;
+  if (OBC_uptime_32 > 655350) {
```

Comment:
Will this ever return true?

Reply:
I don't see why it couldn't. But this seems like a weird way to do things. @nsorense4 do you know why it's written like this?